README.md

```diff
@@ -1,11 +1,272 @@
-# COSS
+# COSS (beta)
 
-(this repo is a work in progress)
+A great way to deal with a large CSS codebase.
 
 __Component Oriented Style Sheets__
 
-While building a large-scale application, it is really important to structure the css code in a maintainable way. But large-scale applications were once small applications and managing, mantaining and structuring the the css was not a problem. There are Methodologies for building css like BEM, OOCSS, SMACSS or Atomic. All of them are really helful if implemented since the beginning but they are not really handy when the css has no structure and counts more than 10k lines.
+While building a large-scale application, it is really important to structure the CSS code in a maintainable way. But large-scale applications were once small applications and managing, mantaining and structuring the CSS was not a problem. There are methodologies for building CSS like BEM, OOCSS, SMACSS or Atomic. All of them are really helpful if implemented since the beginning but they are not really handy when it comes to CSS with no structure and counts more than 30 0000 lines.
```

Comment:
when it comes to CSS without structure counting more than 30 0000 lines of code.